enos/modules/test_e2e/main.tf

```diff
@@ -279,6 +289,8 @@ resource "enos_local_exec" "run_e2e_test" {
     E2E_CLIENT_USERNAME                          = var.client_username
     E2E_CLIENT_PASSWORD                          = var.client_password
     E2E_CLIENT_TEST_DIR                          = var.client_test_dir
+    E2E_CLIENT_VERSION                           = var.client_version
+    E2E_SERVER_VERSION                           = var.server_version
```

Comment:
nit: What do you think about being more specific here? Also groups these related variables together.
```
E2E_TARGET_RDP_SERVER_VERSION
```